site/content/userguide/usage/branches-and-ci.md

```diff
@@ -5,7 +5,10 @@ weight: 25
 
 You have created jsonnet components, a `qbec.yaml` file and declared a dev environment. These are
 good enough for local development and/ or commits to a master branch. Now you are looking for a
-way to create objects with slightly different names as part of functional tests of your pull request builds. 
+way to create objects with slightly different names as part of functional tests of your pull request builds.
+
```

Comment:
still doesn't fit IMO in this page. I propose you leave it out and we can figure out how to update the docs later. The help you added to the command line is good enough for now.